core/src/main/kotlin/event/channel/ChannelUpdateEvent.kt

```diff
@@ -45,6 +74,15 @@ public class NewsChannelUpdateEvent(
     }
 }
 
+/**
+ * The event dispatched when a [TextChannel] is updated.
+ *
+ * The [TextChannel] may have been turned into a [NewsChannel].
+ *
+ * The [old] [TextChannel] may be null unless it has been stored in the cache.
```

Comment:
This will render weird in the dokka output:

The [old](https://kordlib.github.io/kord/core/dev.kord.core.entity.channel/-text-channel/index.html) may be null unless it has been stored in the cache.

You can use this as a workaround:
``The [old][old] [TextChannel] may be `null` unless it has been stored in the cache.``